api-reference/media-api/list-and-search-files.md

```diff
@@ -1012,6 +1061,21 @@ resp, err := ik.Media.Files(ctx, media.FilesParam{
 })
 ```
 {% endtab %}
+{% tab title=".Net" %}
+```.net
+var imagekit = new ImageKit({
+        publicKey : "your_public_api_key",
+        privateKey : "your_private_api_key",
+        urlEndpoint : "https://ik.imagekit.io/your_imagekit_id/"
+});
+
+GetFileListRequest model = new GetFileListRequest
+    { 
+        SearchQuery = "tags = \"tag1,tag2\"",
```

Comment:
What is `SearchQuery = "tags = \"tag1,tag2\"",`?

Incorrect example and values. In this we are not using `searchQuery`. Instead we are using `Tags` directly. Also values are inconsistent with other examples, it should be `"Tags = 'sale, summer'"`.